src/backend/distributed/planner/distributed_planner.c

```diff
@@ -527,6 +527,31 @@ GetRTEIdentity(RangeTblEntry *rte)
 }
 
 
+/*
+ * GetQueryLockMode returns the necessary lock mode to be acquired for given
+ * query according to "rellockmode" field of RangeTblEntry struct from
+ * parsenodes.h in PostgreSQL 12.x.
```

Comment:
I think you can drop ` in PostgreSQL 12.x.` part here as this will be outdated soon and do not provide any real value IMO

Reply:
makes sense, I will address this one tomorrow